server/sandbox_run.go

```diff
@@ -544,6 +544,20 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 		return nil, err
 	}
 
+	defer func() {
+		if err != nil {
+			// Clean-up steps from RemovePodSanbox
+			timeout := int64(10)
+			if err2 := s.Runtime().StopContainer(ctx, container, timeout); err2 != nil {
+				logrus.Warnf("failed to stop container %s: %v", container.Name(), err2)
+			}
```

Comment:
could we backport 334ff802cf42948389914ab0a113069910d072c6 as well so we have the definition for `.WaitContainerStateStopped`?

Reply:
yup